src/main/java/com/kappacrypto/Clients/Twitter.java

```diff
@@ -27,13 +27,11 @@
 
     public Twitter() {
         TwitterCredentials credentials = TwitterCredentials.builder()
-                .apiKey("NDXoF8Dh9BkVGkMjK1P9uRRoe")
-                .apiSecretKey("1e52QYIGD7pgkRTkBAsJeExTqjcUV2p9ZoUZFW5JJeGnAybuxx")
-                .bearerToken("AAAAAAAAAAAAAAAAAAAAAEbpXgEAAAAAeXhSEgWJFIs9rZVBpiO4wYqc5YI%3DbUlUgMo804gy02mwA1K2ieRKB5D8a4cuMbDpGiByIB8uhJZzNi")
```

Comment:
Accidentally pushed on initial commit, these have been removed and are stored securely